svm/src/lib.rs

```diff
@@ -220,6 +229,13 @@ impl LiteSVM {
         self
     }
 
+    /// Sets the Geyser plugin.
+    #[cfg(feature = "geyser-plugin")]
+    pub fn with_geyser_plugin(mut self, plugin: Box<dyn GeyserPlugin>) -> Self {
```

Comment:
I don't know if it's easier to put another with or to deal with the svm with a decorator pattern or something similar if after we plan to add other things